tt_fabric/CMakeLists.txt

```
@@ -0,0 +1,61 @@
+#set(TT_FABRIC_OBJECTS
+#    ${CMAKE_CURRENT_SOURCE_DIR}/control_plane.cpp
+#    ${CMAKE_CURRENT_SOURCE_DIR}/routing_table_generator.cpp
+#    ${CMAKE_CURRENT_SOURCE_DIR}/mesh_graph.cpp
+#)
+
+#add_library(tt_fabric ${TT_FABRIC_OBJECTS})
+
+add_library(tt_fabric)
+add_library(Metalium::Fabric ALIAS tt_fabric)
```

I started this nomenclature based on the assumption that Metal == Metalium.

Since recently finding a doc that spells out the (current) names for everything, I now come to understand that "tt-metal" repo is actually "TT-Metalium And Friends".

To reflect that, I think this would be more in line with intended names:

```suggestion
add_library(TT::Fabric ALIAS tt_fabric)
```

---

I can make this change.